environments/sbox/sbox.tfvars

```diff
@@ -677,13 +677,22 @@ frontends = [
     shutter_app      = false
     backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-sandbox-platform-hmcts-net"
-  }
+  },
+  {
+    product          = "labs-goldenpath-connor-okane"
+    name             = "labs-goldenpath-connor-okane"
+    custom_domain    = "labs-goldenpath-connor-okane.sandbox.platform.hmcts.net"
+    dns_zone_name    = "sandbox.platform.hmcts.net"
+    backend_domain   = ["firewall-sbox-int-palo-labs-goldenpath-connor-okane.uksouth.cloudapp.azure.com/"]
+    certificate_name = "wildcard-sandbox-platform-hmcts-net"
+    disabled_rules   = {}
+  },
 ]
 
 apim_appgw_exclusions = [
   {
     match_variable = "RequestArgNames"
     operator       = "Equals"
     selector       = "iss"
-  }
-]
+  },
```

Comment:
```suggestion
  }
```

Reply:
Thanks for spotting! Let me go and update that now